dev/com.ibm.ws.kernel.service.location/src/com/ibm/ws/kernel/service/location/internal/SymbolicRootResource.java

```diff
@@ -270,7 +270,7 @@ public URI toExternalURI() {
     @Override
     @Trivial
     public File asFile() {
-        return null;
+        return root;
```

Comment:
Is there a test for this?

Reply:
Added a unit test in com.ibm.ws.kernel.service.location/test/com/ibm/ws/kernel/service/location/internal/SymbolicRootResourceTest.java